### DISCUSSION TITLE
Add standard test reports for UnitTesting (Junit, csv, html, ...)

### DISCUSSION THREAD

**Discussion by AlexisTonneau** [Kong/insomnia#4805] [Ideas]
Right now, we can only see test reports from console output. We can change the reporter, but it's only limited to a console output

It would be super useful to retrieve JUnit XML test results, to be compliant with CI/CD tools.

**Comment by filfreire** [comment]
Hi @AlexisTonneau, thanks for opening this discussion!

Internally we've been listing a ton of improvements we want to do on Insomnia's Test feature, and this is one of them. 

We'll update you folks here if there's any changes or improvements in the future.

In the meantime feel free to report any other issues or open discussions 👍

**Comment by me-nkr** [comment]
is this about giving a detailed test result that includes diff between expected and received result?
